<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SIMULAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>500</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>PAUSE</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>711</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>90</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Datos del planeta</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>170</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Poscición Inicial y</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
     <property name="minimum">
      <double>-100000000000000002021887912715594698857609632321435774113777685620800400499816430935869782753280.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100000000000000002021887912715594698857609632321435774113777685620800400499816430935869782753280.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>440</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>210</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Velocidad inicial x</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
     <property name="minimum">
      <double>-10000000000000000159028911097599180468360808563945281389781327557747838772170381060813469985856815104.000000000000000</double>
     </property>
     <property name="maximum">
      <double>99999999999999995164818811802792197885196090803013355167206819763650035712.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>¿Cuantos planetas quiere colocar?</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>130</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Poscición Inicial x</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
     <property name="minimum">
      <double>-999999999999999926539781176481198923508803215199467887262646419780362305536.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000000000000043377296974619186073290293324951939311791773789336116812889681110941323755520.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_7">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>330</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Radio</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_7">
     <property name="minimum">
      <double>-10000000000000000159028911097599180468360808563945281389781327557747838772170381060813469985856815104.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999999999999999549291066784979473595300225087383524118479625982517885450291174622154390152298057300868772377386949310916067328.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_6">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>290</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Masa</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_6">
     <property name="minimum">
      <double>-10000000000000000303786028427003666890752.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000000000000000159028911097599180468360808563945281389781327557747838772170381060813469985856815104.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_5">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>250</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Velocidad inicial y</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
     <property name="minimum">
      <double>-10000000000000000213204190094543968723012578712679649467743338496.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999999999999956819772641641815758405104477258378281795396215622882607621111488153942930947432322044748890112.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>50</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CONFIRMAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>749</x>
      <y>500</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>LIMPIAR</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>360</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Rojo</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>400</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Amarillo</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>380</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Azul</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>380</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Verde</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_5">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>360</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Gris</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_6">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>400</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Negro</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
